app/components/formatted_time.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import moment from 'moment-timezone';
+import React from 'react';
+import {Text, TextProps} from 'react-native';
+
+import type {UserTimezone} from '@mm-redux/types/users';
+
+type FormattedTimeProps = TextProps & {
+    hour12: boolean;
```

Comment:
For boolean constants/flags, do you want to adopt the 'is/has' prefix ?  So instead of hour12 => isHour12 ?

Reply:
Interesting thought, I'm 0/5 on this, what about can/should/could etc?